src/App.jsx

```diff
@@ -1,10 +1,12 @@
 import './App.css';
 import Message from "./Message";
+import TextQuotes from './TextQuotes';
 
 function App() {
   return (
     <> 
-      <Message/> 
+      <Message/>
```

Comment:
It looks like now both the Message component and TextQuotes component are displayed simultaneously, while we probably only want one. Perhaps this is what you're already working on, otherwise I would suggest removing one or combining them in some way.